src/core/issues-notes.js

```diff
@@ -22,12 +22,15 @@ export const name = "core/issues-notes";
 const localizationStrings = {
   en: {
     issue_summary: "Issue Summary",
+    no_issues_in_spec: "There are no issues listed in this specification.",
   },
   nl: {
     issue_summary: "Lijst met issues",
+    no_issues_in_spec: "No hay problemas enumerados en esta especificación.",
   },
   es: {
     issue_summary: "Resumen de la cuestión",
+    no_issues_in_spec: "Er zijn geen problemen vermeld in deze specificatie.",
```

Comment:
Please be sure @thijsbrentjens verifies this translation. Using auto-translations is always a big risk.